src/app/router.js

```diff
@@ -43,6 +43,8 @@
 
   function getDefaultController(dataKey) {
     return function (
+                      $localForage,
+                      $timeout,
                       sectionsService,
                       sidenavService,
                       userService
```

Comment:
$controller will not work due to Configuration blocks - get executed during the provider registrations and configuration phase. Only providers and constants can be injected into configuration blocks. This is to prevent accidental instantiation of services before they have been fully configured.